src/redist/targets/GenerateLayout.targets

```diff
@@ -3,8 +3,6 @@
     <RedistLayoutPath>$(BaseOutputPath)$(Configuration)\dotnet\</RedistLayoutPath>
     <SdkInternalLayoutPath>$(BaseOutputPath)$(Configuration)\dotnet-internal\</SdkInternalLayoutPath>
     <DownloadsFolder>$(IntermediateOutputPath)downloads\</DownloadsFolder>
-    <!-- add -internal to use the internal builds, except for Mac which already uses internal builds -->
```

Comment:
Do we have to do something else so that Mac starts using the non-internal packages?

Reply:
Windows-only at the moment. If you try to grab any PGO packages from Mac it will hard error.

Mac is third on my priority list after LInux. Tbh if clang on Mac can use the Linux clang PGO data it may never be worth it to train on Mac specifically.